lib/ansible/modules/network/aci/aci_aaa_user.py

```diff
@@ -307,6 +314,7 @@ def main():
     last_name = module.params.get('last_name')
     phone = module.params.get('phone')
     state = module.params.get('state')
+    nameAlias = module.params.get('name_alias')
```

Comment:
Internally we use snake case in python, this related to common practices in python.
Another good practice is to keep the style within a file consistent, which this change breaks as well.
So I am not a big fan of mixing styles like this.

Reply:
@nkatarmal-crest